ls1intum/Artemis
# Reduce space taken by navbar and notification panels

### Checklist

### Motivation and Context

The margins of the notification panels were a bit too large, which was highlighted even more due to the recent breadcrumb changes.
Before:
![grafik](https://user-images.githubusercontent.com/72132281/101155312-36720a80-3627-11eb-91bb-dd33122143a2.png)
![grafik](https://user-images.githubusercontent.com/72132281/101155346-425dcc80-3627-11eb-8ae9-9261a7b3d8ea.png)
After:
![grafik](https://user-images.githubusercontent.com/72132281/101155361-48ec4400-3627-11eb-817b-6da699711a4a.png)
![grafik](https://user-images.githubusercontent.com/72132281/101155358-47228080-3627-11eb-9659-3cb37905642e.png)
![grafik](https://user-images.githubusercontent.com/72132281/101155363-4d186180-3627-11eb-99d0-1fc23526d579.png)

### Steps for Testing

1. Log in to Artemis
2. Create a new System Notification ("Server Management" > "System Notifications")
3. Check if the margins fit in both a student's course overview and the instructors' course management view ("Course Overview" and "Course Management"; in the latter you have breadcrumbs)
